.github/workflows/_buildpacks-release.yml

```diff
@@ -94,62 +111,31 @@ jobs:
-        run: cargo libcnb package --release --package-dir ${{ env.PACKAGE_DIR }}
-
-      - name: Generate buildpack matrix
-        id: generate-buildpack-matrix
-        run: actions generate-buildpack-matrix --package-dir ${{ env.PACKAGE_DIR }}
-
-      - name: Generate changelog
-        id: generate-changelog
-        run: actions generate-changelog --version ${{ steps.generate-buildpack-matrix.outputs.version }}
-
-      - name: Temporary fix for bash-based buildpacks
+        env:
+          BUILDPACKS: ${{ steps.generate-buildpack-matrix.outputs.buildpacks }}
         run: |
-          buildpacks='${{ steps.generate-buildpack-matrix.outputs.buildpacks }}'
-
-          bash_buildpack_source_dirs=()
-          bash_buildpack_output_dirs=()
-
-          # copy any bash-based buildpack to target buildpack dir because `cargo libcnb package` will ignore them
-          for buildpack in $(jq --exit-status -c '.[]' <<< "${buildpacks}"); do
+          package_dir=$(realpath "${{ env.PACKAGE_DIR }}")
+          for buildpack in $(jq --exit-status -c '.[]' <<< "${BUILDPACKS}"); do
             buildpack_dir=$(jq --exit-status -r '.buildpack_dir' <<< "${buildpack}")
-            output_dir=$(jq --exit-status -r '.buildpack_output_dir' <<< "${buildpack}")
-            if [ ! -d "${output_dir}" ]; then
-              echo "bash-based buildpack detected at ${buildpack_dir}"
-              cp -R "${buildpack_dir}" "${output_dir}"
-              bash_buildpack_source_dirs+=("${buildpack_dir}")
-              bash_buildpack_output_dirs+=("${output_dir}")
-            fi
-          done
-
-          # replace dependencies that reference a bash-buildpack
-          for buildpack in $(jq --exit-status -c '.[]' <<< "${buildpacks}"); do
-            output_dir=$(jq --exit-status -r '.buildpack_output_dir' <<< "${buildpack}")
-            echo "checking dependencies in ${output_dir}/package.toml"
-            for dep in $(yq -oy '.dependencies[].uri' "${output_dir}/package.toml"); do
-              if realpath "${dep}" &> /dev/null; then
-                dep_path=$(realpath "${dep}")
-                for i in "${!bash_buildpack_source_dirs[@]}"; do
-                  bash_buildpack_source_dir="${bash_buildpack_source_dirs[$i]}"
-                  bash_buildpack_output_dir="${bash_buildpack_output_dirs[$i]}"
-                  if [ "${bash_buildpack_source_dir}" = "${dep_path}" ]; then
-                    echo "replacing ${dep} with ${bash_buildpack_output_dir}"
-                    sed -i 's|'"$dep"'|'"$bash_buildpack_output_dir"'|g' "${output_dir}/package.toml"
-                  fi
-                done
+            cd "$buildpack_dir"
+            for target in $(jq --exit-status -c '.targets | .[]' <<< "${buildpack}"); do
+              output_dir=$(jq --exit-status -r '.output_dir' <<< "${target}")
+              if [[ "$buildpack_dir" -ef "$output_dir" ]]; then
+                echo "Skipping packaging of ${output_dir}. Not a libcnb.rs buildpack."
```

Comment:
hmm, @joshwlewis just wondering if this is the correct action to take here. We still need to move the bash buildpacks into the `${package_dir}` location because at the end of this Compile Buildpacks job we cache the contents of the `${package_dir}` folder.  Only the `${package_dir}` contents are restored during the publishing steps (we don't do a git checkout) so, however this works, the bash buildpack should be copied into that folder.

Reply:
Hmm, I see. Well, I could restore that code. Or, we could run `git checkout` on the other steps. Neither is as efficient as I'd like.

Another option maybe - restore the copy logic, but instead of rewriting other package.tomls, I could symlink the bash buildpack source directory to it's location in the package dir.

Reply:
Yeah, this annoying script solved 2 problems:
1. persisting the bash buildpack so you have something to package in later jobs
2. rewriting package.toml files that reference the bash buildpack so that composite buildpacks can be packaged

If you can accommodate those two cases via some other mechanism that would be good. Otherwise we'll still need this kludge around for awhile longer.

Reply:
I left the copy in place. I dropped the dependency rewrite stuff, though. Instead, I used the `Checkout` action, but only for composite buildpacks. That way, all buildpacks are in the package dir, and any composite buildpacks can find bash buildpacks in their original spot.